ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -15,6 +15,11 @@ namespace ksp_plugin_adapter {
 public partial class PrincipiaPluginAdapter
     : ScenarioModule,
       WindowRenderer.ManagerInterface {
+      
+  private const String чебышёв_release_name = "Чебышёв";
```

Comment:
Call this `next_release_name`, etc., so that we don't need to rename the identifiers when doing a release.